src/screens/user-management/reducer.test.ts

```diff
@@ -0,0 +1,52 @@
+import { produce } from "immer";
+import reducer, { initialState } from "./reducer";
+import { QueryParamsAction } from "./types";
+
+describe("User Management Reducer", () => {
+  it("should update filters when action type is UPDATE_FILTER", () => {
```

Comment:
Use `context` block when conditions are present.
Ref: https://www.betterspecs.org/#contexts

```typescript
context("when action type is UPDATE_FILTER", () => {
  it("updates name and status filters", () => {
    // ...
  });
})
```

--- 

Again, NO use `should` for test description